packages/typespec-go/test/cadlranch/parameters/spreadgroup/zz_spreadmodel_client.go

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Go Code Generator. DO NOT EDIT.
+
+package spreadgroup
+
+import (
+	"context"
+	"errors"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// SpreadModelClient contains the methods for the Parameters.Spread namespace.
+// Don't use this type directly, use [SpreadClient.NewSpreadModelClient] instead.
+type SpreadModelClient struct {
+	internal *azcore.Client
+}
+
+//   - bodyParameter - This is a simple model.
```

Comment:
Are the missing method names our problem or the tsp's?

Reply:
It's the tsp. A lot of the cadl-ranch tests don't have doc comments for the operations. I agree it looks funny. We _could_ just omit the param docs if the operation doesn't have docs. However, given that we'd never ship a real tsp in this way, I don't feel super-strong about changing it.